### PR TITLE
Feature/develop UI

### PR DESCRIPTION
**Pull request by AckonSamuel**
## Description:
In this milestone, styling is added. Ziplin is used as a collaboration tool.

### General requirements:

- [x]  Make sure that there are no linter errors.
- [x]  Make sure that you used correct flow Gitflow
- [x]  Make sure that you documented your work in a professional way

### Milestone requirements

- [x] Use the information [Zeplin](https://zeplin.io/) to style the project.
- [x] The final design matches Zeplin's design

<hr>
 Happy Reviewing 🐣

### CODE REVIEW

File: src/components/sytles/book.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  align-items: center;
+  margin-right: auto;
+  font-size: 1.7em;
+  gap: 0.3em;
+}
+
+#title {
+  width: 70%;
+  margin: 0.188% 20.188% 0.05em 0;
+  font-family: 'Roboto Slab', serif;
```
Comment (Gambit142) [inline src/components/sytles/book.css]: - [x] (Required) I noticed you are repeating this rule in this file. I would like to advise that you either turn it to a class selector (`.roboto`) and use it where necessary or you make it a CSS variable and use it in the file. 👍

File: src/components/sytles/navbar.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+a {
+  text-decoration: none;
+  font-size: 1.4em;
+}
+
+#logo {
+  width: 20%;
+  margin: 0.375% 0 0.125% 0;
+  font-family: 'Montserrat', sans-serif;
```
Comment (Gambit142) [inline src/components/sytles/navbar.css]: - [x] (Required) I noticed you are repeating this rule a lot of times in this file. I would like to advise that you either turn it to a class selector (`.montserrat`) and use it where necessary or you make it a CSS variable and use it in the file.

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @AckonSamuel 👋, 

Good job so far!

However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- I noticed that when I click the categories link and then click the book link, the items in the book array are duplicated and then rendered. Kindly fix it. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/components/sytles/book.css (hunk 1 above)
Comment (AckonSamuel) [reply src/components/sytles/book.css]: Fixed! 😄

File: src/components/sytles/navbar.css (hunk 2 above)
Comment (AckonSamuel) [reply src/components/sytles/navbar.css]: Fixed! 😄

**Comment by AckonSamuel** [comment]
Thanks for the review!  @Gambit142

**Review by bimbolabuari — APPROVED** [review]
Hi @AckonSamuel, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**